conans/test/util/vcvars_clangcl_test.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# vim: tabstop=8 expandtab shiftwidth=4 softtabstop=4
+
+import platform
+import unittest
+from nose.plugins.attrib import attr
+from conans.model.settings import Settings
+from conans.client.conf import default_settings_yml
+from conans import tools
+
+
+@attr('visual_studio')
+@unittest.skipUnless(platform.system() == "Windows", "Requires Windows")
+class VCVarsClangClTest(unittest.TestCase):
+
+    def test_simple(self):
+        if platform.system() != "Windows":
```

Comment:
remove this, as it is already checked in `skipUnless()`

Reply:
sure

Reply:
done